docs/deployment/k8s/index.mdx

```diff
@@ -1,31 +1,176 @@
 
-To make it easy to try out wasmCloud and/or integrate it with existing deployments, wasmCloud publishes an [official Helm chart][helm-chart].
+On **this** page, you can find documentation for operators wishing to deploy wasmCloud with Kubernetes, including details on prerequisites such as NATS and how to deploy the operator. 
 
-To get started quickly and see usage examples, read the ["Running the Chart" guide][helm-chart-quickstart] on the [chart README][helm-chart-docs].
+## Getting started with the wasmCloud operator
 
-For a complete listing of configuration for the Helm chart, jump straight to the [`values.yaml`][helm-chart-values.yaml].
+The wasmCloud operator can be found on GitHub at [https://github.com/wasmCloud/wasmcloud-operator](https://github.com/wasmCloud/wasmcloud-operator). It uses custom resource definitions (CRDs) to define wasmCloud hosts as Kubernetes resources. 
 
-[helm-chart]: https://github.com/wasmCloud/wasmCloud/blob/main/chart/Chart.yaml
-[helm-chart-docs]: https://github.com/wasmCloud/wasmCloud/tree/main/chart
-[helm-chart-values.yaml]: https://github.com/wasmCloud/wasmCloud/blob/main/chart/values.yaml
-[helm-chart-quickstart]: https://github.com/wasmCloud/wasmCloud/tree/main/chart#running-the-chart
+The `WasmCloudHostConfig` CRD describes the desired state of a set of wasmCloud hosts connected to the same lattice:
 
-### Kubernetes Service Applier
+```yaml
+apiVersion: k8s.wasmcloud.dev/v1alpha1
+kind: WasmCloudHostConfig
+metadata:
+  name: my-wasmcloud-cluster
+spec:
+  # The number of wasmCloud host pods to run
+  hostReplicas: 2
+  # The cluster issuers to use for each host
+  issuers:
+    # This needs to be the public key of a cluster seed generated by
+    # `wash keys gen cluster`
+    - CDK...
+  # The lattice to connect the hosts to
+  lattice: 83a5b52e-17cf-4080-bac8-f844099f142e
+  # Additional labels to apply to the host other than the defaults set in the operator
+  hostLabels:
+    some-label: value
+  # Which wasmCloud version to use
+  version: 0.82.0
+  # The name of a secret in the same namespace that provides the required secrets.
+  secretName: cluster-secrets
+  # Enable the following to run the wasmCloud hosts as a DaemonSet
+  #daemonset: true
+  # The name of the image pull secret to use with wasmCloud hosts so that they
+  # can authenticate to a private registry to pull components.
+  # registryCredentialsSecret: my-registry-secret
+```
 
-Cosmonic has open sourced a basic connector that enables bridging services running in Kubernetes with services running in wasmCloud.
+The CRD requires a Kubernetes Secret with the following keys:
 
-The interface, provider, and component can be found at [on Github under `cosmonic/kubernetes-applier`][k8s-applier].
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-wasmcloud-cluster
+data:
+  # You can generate this with wash:
+  # `wash keys gen cluster`

```

Comment:
This may be the first time a user is seeing `wash`, maybe add a note about installing the wash cli?